### ISSUE TITLE
Any ways to dynamically update helper-elements?

### ISSUE THREAD

**Issue by epatpol** [isteven/angular-multi-select#578]
I have a use case where by default there's all the helper elements associated with the dropdown, but due to some external changes in data, there are some cases where the already rendered dropdown wouldn't need the helper elements like Filter, Select All, Unselect All etc.

This is kinda dynamic as it depends on other options in the forms. Sometimes I want those helper elements to be there, sometimes only the Unselect All (But not the Select All) for instance. Only solution I found was to re-render the component (remove and render a new one) but that seems overkill. I also managed to remove the helper elements with Jquery, but then some code in multiselect breaks the tab navigation, as I think it expects the helper-elements to be there but they're not, so user is stuck in the filter when tabbing.

tl;dr Why doesn't changing the helper-elements from "all" to "none" dynamically update the dropdown like it does with say, translation.

Thanks!
